modules/installation-vsphere-config-yaml.adoc

```diff
@@ -30,9 +35,26 @@ platform:
     password: password <9>
     datacenter: datacenter <10>
     defaultDatastore: datastore <11>
+ifndef::disconnected[]
 pullSecret: '{"auths": ...}' <12>
+endif::disconnected[]
+ifdef::disconnected[]
+pullSecret: '{"auths":{"<bastion_host_name>:5000": {"auth": "<credentials>","email": "you@example.com"}}}' <12>
```

Comment:
In the earlier docs you were talking about appending the mirror creds to the cloud.openshift.com ceds, but this line here is _just_ the mirror creds.  That's fine if the cluster doesn't need to auth to any of the standard Red Hat services, but it's not going to work if there is proxy or whatever access and the cluster is supposed to be pushing Telemetry/Insights, pulling registry.redhat.io images, etc.

Reply:
@wking, do we know which use case is more likely or preferred?

Reply:
I dunno, but sounds like you're planning to address this with a note in a reroll.

Reply:
In the "preparing" topic, I'm saying to use just the mirror creds, so I'm going to keep this reference as-is.